pysatCDAAC/instruments/cosmic_gps.py

```diff
@@ -19,67 +19,78 @@
 name
     'gps' for Radio Occultation profiles
 tag
-    Select profile type, or scintillation, one of:
-    {'ionprf', 'sonprf', 'wetprf', 'atmprf', 'scnlv1'}
-sat_id
+    Select profile type, or scintillation, using one of the following keys:
+       {'ionprf': 'Ionospheric Profiles',
+        'wetprf': 'Atmospheric profiles with moisture',
+        'atmprf': 'Atmospheric profiles without moisture',
+        'eraprf': 'ERA-40 Interim reanalysis data',
+        'gfsprf': 'NCEP operational analysis data',
+        'ionphs': 'Ionospheric excess phase',
+        'podtec': 'Absolute Total Electron Content and auxiliary data',
+        'scnlv1': 'S4 scintillation index and auxiliary data'}
```

Comment:
The spacing here will raise a warning in autodoc.  Everything needs to have the same number of spaces in front of it as "Select".